_posts/2018-10-29-go.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Go over Nabla- App Safety meets Host Isolation
+---
+
+This blog presents how a Go application can be built and run as a Nabla container.
```

Comment:
It may not be clear that `Go` in this context means golang. Maybe we can add a statement saying that we are dealing withe `golang`?